task/net/src/bsp/gimlet_1.rs

```diff
@@ -45,7 +45,7 @@ pub struct Bsp(mgmt::Bsp);
 pub fn preinit() {
     // Wait for the sequencer to turn on the clock
     let seq = Sequencer::from(SEQ.get_task_id());
-    while seq.is_clock_config_loaded().unwrap() == 0 {
+    while seq.is_clock_config_loaded().map(|b| b == 0).unwrap_or(true) {
```

Comment:
Could also: `.unwrap_or(0) == 0` for slightly less code.

Reply:
Thanks, I was wondering if there was a cleaner way to write this!